style.css

```diff
@@ -133,15 +144,20 @@ body {
   justify-content: space-between;
 }
 
-/* Transition button */
 .btn-work {
+  margin-top: 36px;
   width: 77px;
   height: 40px;
   display: flex;
   align-items: center;
```

Comment:
[OPTIONAL]

- [ ] There are currently several occurrences of these two properties. To DRY up your stylesheet, you could have a class with these properties, reuse the class on the elements and specify only unique rules in your selectors. Example:
```
.center {
  display: flex;
  align-items: center;
}
```

Reply:
OK thank you a lot for this shortcut, it's really helpful.